lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -114,15 +116,17 @@ defmodule ArchEthic.Mining.PendingTransactionValidation do
          },
          previous_public_key: previous_public_key
        }) do
-    with {:ok, _, _, _, _, key_certificate} <- Node.decode_transaction_content(content),
+    with {:ok, ip, port, _, _, key_certificate} <- Node.decode_transaction_content(content),
          root_ca_public_key <- Crypto.get_root_ca_public_key(previous_public_key),
+         true <- validate_node?(@validate_node, ip, port, previous_public_key),
```

Comment:
maybe renaming it to `valid_ip?`, so this will be more meaningful

Reply:
We are already using `Networking.validate_ip?(ip)`